pkg/printer/printer.go

```diff
@@ -164,6 +170,69 @@ func printJSONYaml(format OutputFormat, v interface{}) error {
 	return nil
 }
 
+func FormatVariableDescriptions(variableMap map[string]*core.Variable) {
+	keys := make([]string, 0, len(variableMap))
+	// sort the keys for testing and consistency with other output formats
+	for k := range variableMap {
+		keys = append(keys, k)
+	}
+	sort.Strings(keys)
+
+	var descriptions []string
+	//for k, v := range variableMap {
+	for _, k := range keys {
+		if k == DefaultFormattedDescriptionsKey {
```

Comment:
why is this check necessary? shouldn't the variable map just be straight from the entity closure?

Reply:
There're some side effects in the tests that I don't fully understand. For example, if I change the order of the tests, or if I try to run tests individually, things break. This is one of the problems I ran into, the function under test got called twice.

Reply:
@mayitbeegh If any test execution changes based on the order then that would be case if you are using global state across all the tests.
In such case move common state changes in the setup function
and override behavior for test specific states after the setup is called in the test. That should fix such cases.
We have done this across most of the tests in flytectl.
Also is it possible for you to just write unit tests covering this function, the bot is showing that this function is not covered anywhere

Reply:
@pmahindrakar-oss Okay, I can fix that